roles/kafka_connect/tasks/connect_plugins.yml

```diff
@@ -41,6 +41,15 @@
   when: kafka_connect_plugins_remote|length > 0
   notify: restart connect distributed
 
+- name: Set Group and Owner for Remote Plugins
```

Comment:
```suggestion
- name: Set Group and Owner for Plugins
```

Reply:
there are 2 tasks above this which will use this task for setting groups and owners, one for local another for remote